src/main/java/com/exasol/projectkeeper/SurefirePluginPomTemplate.java

```diff
@@ -0,0 +1,24 @@
+package com.exasol.projectkeeper;
+
+import static com.exasol.projectkeeper.AbstractProjectKeeperMojo.MODULE_DEFAULT;
+
+import java.util.Collection;
+
+import org.w3c.dom.Node;
+
+public class SurefirePluginPomTemplate extends AbstractPluginPomTemplate {
```

Comment:
Don't forget to add comments to the public classes and methods. You can have problems with releasing on the MC